CemrgApp/CMakeExternals/VMTK.cmake

```diff
@@ -34,8 +34,8 @@ if(MITK_USE_VMTK)
       )
     endif()
 
-    set(VMTK_GIT_REPOSITORY "https://github.com/vmtk/vmtk.git" CACHE STRING "The git repository for cloning VMTK")
-    set(VMTK_GIT_TAG "v1.4.0" CACHE STRING "The git tag/hash to be used when cloning from VMTK_GIT_REPOSITORY")
+    set(VMTK_GIT_REPOSITORY "https://github.com/CemrgDevelopers/vmtk.git" CACHE STRING "The git repository for cloning VMTK")
```

Comment:
We should document somewhere why we are temporarily using a forked `vmtk` and not the official releases.

Reply:
Do you think in the wiki would suffice or in a comment on this CMake file?